bot/exts/previewing/preview.py

```diff
@@ -37,14 +37,16 @@ async def server_icon(
         icon = (await download_image(file_url)).resize(ICON_SIZE)
         icon = add_background(icon, Preview.background_color(mode.lower()))  # type: ignore
 
-        with Image.open(ICON_TEMPLATES.format(mode=mode)) as template:
-            preview = Image.new("RGBA", template.size)
-            for position in ICON_POSITIONS:
-                preview.paste(icon, position)
+        def long_stuff():
```

Comment:
Let's use a more descriptive name for this function. For example: `process` or `get_preview`.

Reply:
I did *not* mean to commit this.

Reply:
Sounds like 5 am coding 😉